Server/MirEnvir/Map.cs

```diff
@@ -1350,7 +1350,7 @@ private void CompleteMagic(IList<object> data)
                             SpellObject ob = new SpellObject
                                 {
                                     Spell = Spell.PoisonCloud,
-                                    Value = value + bonusdmg,
+                                    Value = value,// + bonusdmg,
```

Comment:
How come Stat.PoisonAttack (bonusDmg) is no longer used as part of PoisonCloud?

Reply:
I am sure that the official green poison damage is (MaxSC+MinSC)/2, but I am not sure if Stat.PoisonAttack is used. According to experience, PoisonAttack should be a bonus to green poison damage. I think the direct damage of poison cloud should not Receives a bonus, but the green poison damage caused by the poison cloud should have a bonus.The code has been modified.